contracts/contracts/child/ChildToken/ChildMintableERC20.sol

```diff
@@ -0,0 +1,79 @@
+pragma solidity 0.6.6;
```

Comment:
is this just moving file or you build a new file?

Reply:
Copy from matic repo